### DISCUSSION TITLE
请问怎么找到一个使用该框架的网站？之前用的网站打不开了

### DISCUSSION THREAD

**Discussion by windy1021** [qd-today/qd#430] [Q&A]
如题

**Comment by windy1021** [comment]
网站打不开了但他好像还在帮我自动签到，，但有些项目无法编辑，这种情况该怎么办

**Reply by a76yyyy** [reply]
你用的谁的网站
建议自己搭建，可以临时用这个
https://qd.a76yyyy.cn

**Reply by windy1021** [reply]
krystal033.tk
这个是之前用的网站的地址，但现在尝试了各种方法都一直打不开，但既然里面的签到脚本还在运行会不会有可能是网站换了域名什么的？我之前有设置推送到tgbot，能否通过tg上面的链接找到推送的网站地址呢？<img src="https://files.catbox.moe/vcylqy.png"  />

**Reply by windy1021** [reply]
搭建我不太会，，比较小白，上午尝试了一下docker，但不太会用🥲

**Reply by windy1021** [reply]
还有一个问题就是用大佬你发的网站注册不了tg推送，我填了前两项，之前那个网站可以成功推送，但是在这里显示推送失败，我加上了使用说明里面的tg.mydomain.com显示连接失败，这要怎么办，感谢大佬

**Reply by a76yyyy** [reply]
tg推送需要自定义代理, 建议使用其他国内推送

**Reply by windy1021** [reply]
那请问下现在这个网站登不上去但签到脚本每天还在运行，我要怎么编辑或者禁用呢😂有办法吗

**Comment by a76yyyy** [comment]
> 那请问下现在这个网站登不上去但签到脚本每天还在运行，我要怎么编辑或者禁用呢😂有办法吗

联系框架站管理员，或者在你的网页里退出账号重新登录让旧的cookie失效

联系方法可以通过给你发的框架验证邮件的发件人获得，如果没有验证邮件就没有办法了

**Reply by windy1021** [reply]
悲。。。我没找到邮件，好像是那种不需要验证的
